### ISSUE TITLE
RTF GUI freezes on opening

### ISSUE THREAD

**Issue by pefreis** [DynamoDS/RevitTestFramework#94]
Hey! I've just started trying RTF but couldn't do much... Every time I launch **RevitTestFrameworkGUI.exe**, it opens up the "Runner" window and freezes imediately.

![RTF_GUI_freezing](https://user-images.githubusercontent.com/42272817/57157094-18428980-6db6-11e9-9074-fe2f63fac5e6.PNG)

I've already tried installing it directly through the setup file at "...\tools\Output\" as well as running the .exe file built from Visual Studio solution, but with no success. The weirdest thing is that if I debug the **RevitTestFrameworkGUI project** more slowly the Revit Test Runner starts seamlessly, which won't occur in case I go through it faster.

Can anyone help me??

**Comment by ZiyunShang** [comment]
Hi @pefreis , I have not seen this problem before and can not reproduced it. Could you give some more information about this issue? For example, the OS version, the version of RTF you are using, and the Revit version.

**Comment by pefreis** [comment]
@ZiyunShang 

OS: Windows 10 (64 bit)
RTF version: 2019
Revit version: 2019.2

Also it is worth mentioning that **RevitTestFramework.exe** seems to be running properly.

**Comment by ZiyunShang** [comment]
Is that RevitTestFramework.exe means RevitTestFrameworkConsole.exe? There should not be a file called RevitTestFramework.exe.
And it seems that when you debug the solution, the RevitTestFrameworkGUI can work well, right? 
Is this issue just occur when you open RevitTestFrameworkGUI.exe manully in release build or installation?

**Comment by pefreis** [comment]
That's right, I have been using **RevitTestFrameworkConsole.exe** for some days without any problem.

The issue with RevitTestFrameworkGUI.exe occurs when:

- Running it manually from installation
- Running it manually from debug build
- Running it manually from release build

It also happens when debugging, in some cases. I will try to explain it better:

When it reaches the breakpoint I show in the image, if I **wait some seconds** before I continue, it will **start properly**. If I **don't wait any time**, or **don't wait enough** (I don't know how long is enough...), it will **start and freeze**, the same way it would occur in the previous cases.

![debug](https://user-images.githubusercontent.com/42272817/57451647-41409f80-7238-11e9-936d-44f20f637e86.PNG)

**Comment by ZiyunShang** [comment]
It seems like you open the RevitTestFrameworkGUI.exe with a parameter. It's no need to add a parameter then open the GUI.exe.
